fileblock/src/simplefs.rs

```diff
@@ -0,0 +1,25 @@
+// slightly arbitrary size of our "disk": 4GIB, broken into 4KiB blocks
+// 4 * 2.pow(30) / 4096 == 2Kib required to store bitmap
+const BITMAP_LEN: usize = 1048576_usize;
```

Comment:
We know when we create the file system format what the size of the block we have to work with is (some value less than the max size of the partition). This value has to be variable based on the number of blocks available on the underlying storage.

Reply:
> This value has to be variable based on the number of blocks available on the underlying storage.


right. in this example our storage is a file that's arbitrarily limited to 4gb in size just cause.

Reply:
Maybe `BITMAP_LIMIT` is more descriptive here since this value represents the upper bound on how much disk it can map to